client/packages/requisitions/src/ResponseRequisition/DetailView/Footer/Footer.tsx

```diff
@@ -29,7 +29,7 @@ export const Footer: FC = () => {
   return (
     <AppFooterPortal
       Content={
-        data ? (
+        data && (
```

Comment:
no functional change - just making it consistent with the request requisition version